packages/css/src/fieldset.css

```diff
@@ -14,7 +14,7 @@
   }
 
   /* Stack everything that is not directly after legend */
-  & > * + * {
+  & > *:not(.ds-sr-only) + *:not(p) {
```

Comment:
Do we need to change this? Would it not be enough to later in the css set margin to 0?

Reply:
Yes, simplified. (i forgot that `:not(selector)`) give same specificity as `selector`